toomb1r/TeamProjects2_Team6
# Emergency Stop command

Issue will be created to set up interrupts for this GPIO pin

# Description

This change will create the emergency stop code for the Emergency Stop button. If the Emergency Stop button is pressed it will send an encrypted signal to PEAT to force it to cease all functions.

Fixes no existing issue

# Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### Unfortunately at this moment we are unable to test this function as it uses RPi.GPIO. This means it can be only tested on a Raspberry Pi.

    - Place a print statement inside the `emergencyStop` command printing out `signal`
    - Run the code
      - This should encrypt the signal and send it to PEAT
    - The output should be the string `stop` 

**Test Configuration**:
* Firmware version:
* Hardware: Raspberry Pi
* Toolchain:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project (Pep 8)
